unittests/util/TestObject.h

```diff
@@ -0,0 +1,135 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of clio: https://github.com/XRPLF/clio
+    Copyright (c) 2022, the clio developers.
+
+    Permission to use, copy, modify, and distribute this software for any
+    purpose with or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL,  DIRECT,  INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#pragma once
+#include <ripple/ledger/ReadView.h>
+#include <string>
+
+/**
+ * Create AccountID object with string
+ */
+[[nodiscard]] ripple::AccountID
+GetAccountIDWithString(std::string id);
+
+/**
+ * Create a simple ledgerInfo object with only hash and seq
+ */
+[[nodiscard]] ripple::LedgerInfo
+CreateLedgerInfo(const char* ledgerHash, ripple::LedgerIndex seq);
```

Comment:
if using `string_view` is an option, we should prefer it over `const char*`. please consider here and everywhere else 👍